# VelocityData.msg
float32 angular_speed_left
float32 angular_speed_right
float32 servo_angle